CRM/Utils/DeprecatedUtils.php

```diff
@@ -600,8 +600,8 @@ function _civicrm_api3_deprecated_validate_formatted_contact(&$params) {
             CRM_Utils_Array::value('value', $value)
           );
           if (!$valid && $value['is_required']) {
-            return civicrm_api3_create_error('Invalid value for custom field \'' .
-              CRM_Utils_Array::value('name', $custom) . '\''
+            throw new CRM_Core_Exception('Invalid value for custom field \'' .
+              $custom['name'] . '\''
```

Comment:
So in theory it could have returned NULL here right?

Reply:
@seamuslee001 yeah - there is some right this would give an e-notice but it seems like our metadata is more consistent now so less of this belt & braces handling is better